### PR TITLE
SSL for nginx, safer supervisord task, VIRTUAL_ENV + other env vars for Galaxy handler

### PR DESCRIPTION
**Pull request by pvanheus**
* Add support for nginx SSL config
* Make supervisord task ignore errors when disabled absent services
* Add VIRTUAL_ENV to galaxy supervisord config
* Add galaxy_handler_env var to allow extra environment vars for the handler

### CODE REVIEW

**Comment by jmchilton** [comment]
:+1:

**Comment by bgruening** [comment]
I'm a little bit worried about the ignore_error changes, because for other with this service installed, it might be useful. Could be use something like `failed_when` and filter out your specific error and only hide this one?

File: tasks/supervisor.yml [hunk 1]
```diff
@@ -37,24 +40,28 @@
...
     - postgresql
   when: supervisor_manage_postgres
+  ignore_errors: yes # ignore error that occurs when service is not installed
 
 - name: Stop and remove proftpd.
   service: name={{ item }} state=stopped enabled=no
   with_items:
     - proftpd
   when: galaxy_extras_config_proftpd and supervisor_manage_proftp
+  ignore_errors: yes # ignore error that occurs when service is not installed
```
Comment (mvdbeek) [inline tasks/supervisor.yml]: @pvanheus would it be sufficient to check if galaxy_extras_install packages was yes?
when: galaxy_extras_config_proftpd and supervisor_manage_proftp and galaxy_extras_install_packages. Or alternatively, might there be a ` | bool` required to get proper evaluation of `galaxy_extras_config_proftpd` ?

**Comment by pvanheus** [comment]
I think this PR needs to be split into its component parts and resubmitted.

**Comment by mvdbeek** [comment]
I'll go ahead and close this PR, I think #103 covers SSL.
@pvanheus Feel free to open another PR or issue if there are other problems.

**Pull request closed without merging**